docs/versioned_docs/version-zenith/partials/_google_cloud_sdk_credentials_setup.mdx

```diff
@@ -0,0 +1,7 @@
+Create an environment variable named `GOOGLE_CREDENTIALS` and set it to the contents of the service account JSON key file. This environment variable is then passed to the Dagger Module as an argument, for use in the Google Cloud Run client library during the client authentication process.
```

Comment:
This isn't necessary. Secrets in `dagger call` can be loaded from files directly:

```
dagger call deploy --credential=file:path/to/service/account-key-file.json
```

Reply:
That makes sense but I feel like we should differentiate these two cases in the docs. When I'm working locally and have the file available I really don't want to have to export the secret to an env if I don't have to (not ideal for security). In GH actions it does make sense to just use an env since that's how GHA does secrets.

Reply:
(Also, the above is not a blocker IMO, can easily be adjusted in a follow-up if needed since the current content is not wrong per-se)